src/main.cpp

```diff
@@ -83,7 +83,7 @@ void ShowLicenseInfo()
 QStringList g_qtFlags = {
   "--disable-web-security",
   "--enable-gpu-rasterization",
-#ifdef Q_OS_LINUX
+#if defined(Q_OS_LINUX) || defined(Q_OS_FreeBSD)
```

Comment:
Does FreeBSD use Wayland? If not leave this as Linux only, it's a workaround that has disadvantages.

Reply:
yes FreeBSD has wayland support

Reply:
@bapt There's a merge conflict here due to `--disable-gpu` being removed in https://github.com/jellyfin/jellyfin-media-player/commit/2a3dadae. 

I'm not sure if it's necessecary on FreeBSD, but it's worth noting that the `--disable-gpu` CLI flag for jellyfinmediaplayer has the same effect.

Please resolve the conflict, but I leave it up to you if you want to hardcode `--disable-gpu` exclusively for FreeBSD.

Reply:
thanks for the heads up, fixed.